solidity/random-beacon/README.adoc

```diff
@@ -68,15 +68,14 @@ malicious calling `RandomBeacon.challengeDkgResult(DKG.Result calldata dkgResult
 function. A malicious DKG result may contain corrupted data, group members not
 selected by the pool, or incorrect supporting signatures. If such malicious
 result is submitted and challenged, the result submitter gets slashed and the
-malicious result is immediately discarded. The length of the challenge period
-and slashing amount are governable parameters.
+malicious result is immediately discarded.
 
 Once the challenge period passes, and no challenges are reported,
 the DKG result submitter should unlock the sortition pool and mark the DKG result as
 accepted calling
 `RandomBeacon.approveDkgResult(DKG.Result calldata dkgResult)` to receive a
 reward. In case the submitter does not call the approve function within a
-specific governable number of blocks, anyone can do that and receive the
+specific number of blocks, anyone can do that and receive the
```

Comment:
The period after which everyone becomes eligible to call the approve function will be governable, right? Isn't it missing it from the list?